config.json

```diff
@@ -0,0 +1,6 @@
+{
+	"hashes_dot_org":"test",
+	"sth_api":"test",
+	"hashcat_exe_name":"hashcat",
+	"hashcat_folder":""
+}
```

Comment:
Should this not be in `APP_DIRS` and not hard-coded into the program? 
https://pypi.org/project/appdirs/